Adds file format description (#18)
CathIAS/TLIO: README.md

The generation of `data.hdf5` is specified in `gen_fb_data.py`, which requires interpolated stamped IMU measurement files and time-aligned VIO states files. The user can generate his/her own dataset with a different procedure to obtain the same fields to be used for network training and filter inputs.

## File formats

### Dataset lists

> `test.txt`, `train.txt` and `val.txt` are all the same. Each row is the name of a sequence to be used for one of the purposes of test, train or validate. The name should be the same of the sequence directory for example `seq1` and `seq22` as above.

Reviewer: ```suggestion
`test.txt`, `train.txt` and `val.txt` are list files specifying the split with one sequence name per row in the testing, training and validation datasets respectively. The name should be the same of the sequence directory for example `seq1` and `seq22` as above.
```

### Used to generate `data.hdf5`

> Timestamps (t) are in microseconds (us). Each row corresponds to a single timestamp. All data is delimited by commas.

Reviewer: ```suggestion
Timestamps (t) are in microseconds (us). Each row corresponds to data in a single timestamp. All data is delimited by commas.
```

> - `my_timestamps_p.txt` VIO timestamps. Used to select data time range.

Reviewer: ```suggestion
- `my_timestamps_p.txt` VIO timestamps.
```

>   - [ t ]

Reviewer: ```suggestion
  - [t]
```

  - Note: single column, skipped first 20 frames

> - `imu_measurements.txt` IMU data, raw and calibrated using VIO calibration

Reviewer: ```suggestion
- `imu_measurements.txt` raw and calibrated IMU data
```

  - [t, acc_raw (3), acc_cal (3), gyr_raw (3), gyr_cal (3), has_vio] 

>   - Note: has been interpolated evenly between images. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file (has_vio==1)

Reviewer: ```suggestion
 - Note: calibration through VIO calibration states. The data has been interpolated evenly between images around 1000Hz. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file (has_vio==1). 
```

> - `evolving_state.txt` VIO evolving states at IMU rate.

Reviewer: ```suggestion
- `evolving_state.txt` ground truth (VIO) states at IMU rate.
```

  - [t, q_wxyz (4), p (3), v (3)]

>   - Note: not interpolated, raw IMU timestamps. Every timestamp in my_timestamps_p.txt will have a corresponding timestamp in this file

Reviewer: ```suggestion
  - Note: VIO state estimates with IMU integration. Timestamps are from raw IMU measurements.
```

> - `calib_state.txt` VIO calibration states at image rate (used in data_io.py)

Reviewer: ```suggestion
- `calib_state.txt` VIO calibration states at image rate (used in `data_io.py`)
```

  - [t, acc_scale_inv (9), gyr_scale_inv (9), gyro_g_sense (9), b_acc (3), b_gyr (3)]
  - Note: Changing calibration states from VIO.
- `atttitude.txt` AHRS attitude from IMU

>   - [ t, qw, qx, qy, qz ]

Reviewer: ```suggestion
  - [t, qw, qx, qy, qz]
```
